tests/test_utils.py

```diff
@@ -8,10 +8,10 @@
 class UtilsGetLocalFileContentTestCase(unittest.TestCase):
     def test_unicode_file_content(self):
         content = utils.get_local_file_content("tests/content_unicode.txt")
-        self.assertTrue(isinstance(content, unicode))
+        self.assertTrue(isinstance(content, str))
```

Comment:
Got it working with https://github.com/randsleadershipslack/destalinator/pull/100/commits/8d9cfe30e9fe38e23bac4c4e3d56f67a240f0e80